selftests/functional/test_skiptests.py

```diff
@@ -32,15 +33,16 @@ def test2(self):
     def test3(self):
         pass
 
-    @avocado.skipIf(False)
+    @avocado.skipIf(False, "won't be skipped")
     def test4(self):
         pass
 
-    @avocado.skipUnless(True)
+    @avocado.skipUnless(True, "won't be skipped")
     def test5(self):
         pass
 
-    @avocado.skip()
+    @unittest.skipIf(True, "Skipped using unittest's skipIf which works"
+                     "but is not recommended.")
```

Comment:
The fact that this works is a limiting factor for future improvements in our skip decorators.